src/style.css

```
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc),
+    url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+.inpt {
+  position: relative;
+  border-width: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.sbb {
+  font-size: 16px;
+  line-height: 22px;
+  top: 20px;
+  left: 40px;
+}
+
+.sbs {
+  font-size: 14px;
+  line-height: 19px;
+  top: 10px;
+  left: 20px;
+}
+
+.inpt:focus {
+  outline: 0;
+  outline-offset: 0;
+  background: transparent;
+}
+
+.big {
+  width: 100vw;
+  height: 70px;
```

```suggestion
  width: 100%;
  height: 70px;
```
do not use height explicitly, you can reach needed hight using paddings